documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
-    "angle": 0, // the heading, in degrees, that the wind is coming from
-    "speed": 3  // the speed, in knots, of the wind
+  "icao": "KSFO",                           // Uppercase ICAO airport code.
+  "iata": "SFO",                            // Uppercase IATA airport code.
+  "magnetic_north": 13.7,                   // Magnetic deviation from Magnetic North, in degrees East. (can be a negative number)
+  "ctr_radius": 80,                         // Radius of the airspace from the center of the airport. (in nautical miles)
+  "ctr_ceiling": 10000,                     // Height of the airspace. (in feet)
+  "initial_alt": 5000,                      // Set altitude for departures to climb if given "clearance as filed", but without a "climb via SID" clearance or altitude assignment.  
+  "position": ["lat", "lon", "elevation"],  // The center of the airport. (in latitude, longitude, and elevation) // See comments below.
+  "rr_radius_nm": 5,                        // Radius of the ring sections. (in nautical miles)
+  "rr_center": ["lat", "lon"],              // The center position of the rings. (in nautical miles)
+  "has_terrain": true,                      // Whether or not the airport has an associated GeoJSON terrain file in "assets/airports/terrain". (true or false)
+  "wind": {                                 // Wind is used for score. // It can affect an aircraft's ground tracks if enabled in settings.
+    "angle": 0,                             // Wind direction. (in degrees)
+    "speed": 3                              // Wind speed. (in knots)
   },
-  "fixes": {
-    "FIXNAME", ["lat", "lon"] // the position, in GPS coordinates, of the fix
+  "fixes": {                                // Locations on the map that aircraft use to travel in and out of the map.
+    "FIXNAME", ["lat", "lon"],              // The name and position of the fix. (in latitude and longitude)
+    "FIXNAME2", ["lat", "lon"]              // You can add as many as you like.
   },
   "runways": [
     {
-      "name":        ["36", "18"],     // the name of each end of the runway
-      "name_offset": [[0, 0], [0, 0]], // the offset, in km, of the runway text when drawn on the map
-      "end":         [                 // the ends of the runway
-                       ["lat", "lon", "elevation"],
-                       ["lat", "lon", "elevation"]
-                     ],
-      "delay":       [2, 2],           // the number of seconds it takes to taxi to the end of the runway
-      "sepFromAdjacent": [2.5, 2.5],   // Distance in nautical miles that another aircraft can come while on parallel approach paths, a violation will occur at 85% of this value
-      "ils":         [true, false]     // not used yet; indicates whether or not that end of the runway has ILS
+      "name": ["36", "18"],                 // Name of the runways.
+      "name_offset": [[0, 0], [0, 0]],      // The offset of the runway label. (in km)
```

Comment:
And with things like this that are rarely used, we should probably add some sort of indication that it is an _optional_ property, that should only be used if it is necessary.